kong/plugins/response-ratelimiting/header_filter.lua

```diff
@@ -25,6 +25,8 @@ local function parse_header(header_value, limits)
 end
 
 function _M.execute(conf)
+  ngx.ctx.increments = {}
```

Comment:
Fixes an edge case when the database is down.

 Marco Palladino | CTO @ Mashape | mashape.com | +1 (415) 361-3858

> On May 6, 2016, at 12:29 PM, Thibault Charbonnier <notifications@github.com> wrote:
> 
> In kong/plugins/response-ratelimiting/header_filter.lua:
> 
> > @@ -25,6 +25,8 @@ local function parse_header(header_value, limits)
> >  end
> >  
> >  function _M.execute(conf)
> > +  ngx.ctx.increments = {}
> ?
> 
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly or view it on GitHub
>